datasets/standvastgoed/vastgoed_0.0.4.json

```diff
@@ -0,0 +1,477 @@
+{
+  "type": "dataset",
+  "id": "standVastgoed",
+  "version": "0.0.4",
+  "status": "beschikbaar",
+  "auth": "FP/MDW",
+  "title": "stand van de vastgoedgegevens",
+  "description": "stand op één bepaalde peildatum van gegevens in het vastgoed domein",
+  "publisher": "Onderzoek en Statistiek, team Basisstatistiek",
+  "contactPoint" : {
+    "vcard:fn" : "Basisstatistiek",
```

Comment:
`contactPoint` is a dict with keys `name` and `email`, see [here](https://schemas.data.amsterdam.nl/docs/ams-schema-spec.html#typedefdef-contact)